dashboard/src/components/api/recipe/che-recipe.factory.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2015-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+'use strict';
+
+interface IRecipeResource<T> extends ng.resource.IResourceClass<T> {
+  create(recipe: che.IRecipe): ng.resource.IResource<T>;
+  getRecipes(): ng.resource.IResource<T>;
+}
+
+/**
+ * This class is handling the recipes retrieval
+ * It sets to the array of recipes
+ * @author Florent Benoit
+ */
+export class CheRecipe {
```

Comment:
on API side recipe service was removed - can be removed here as well.

Reply:
done